cypress/integration/t-2.js

```diff
@@ -0,0 +1,97 @@
+describe('Sourcebooks login with my user and different roles', function() {
+
+    it('Should be able to login with my user with role admin', function () {
+
+        cy.visit('/');
+        cy.get('[id="loginForm.userId"]').click({force:true});
+        cy.get('[aria-label="Jelena Černyšova"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="Admin"]').click();
+        cy.get('[type="submit"]').click();
+
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('Jelena Cernyšova')
+        cy.get('.main-nav').find('li').should('have.length', 6)
+        cy.get('a[class="main-nav__link main-nav__link--active"]').contains('Time Logging')
```

Comment:
This should be updated to `cy.get('.main-nav').find('li')`, so that we can get the whole navigation bar and its items